.github/workflows/deploy-docs-openapi.yml

```diff
@@ -0,0 +1,18 @@
+# This GitHub Actions workflow was auto-generated by the `rdme` cli on 2024-06-25T13:33:11.644Z
+# You can view our full documentation here: https://docs.readme.com/docs/rdme
+name: Deploy OpenAPI Docs
```

Comment:
Do we need to run it every time, or can we limit it to only the subset of files that have changed?

Reply:
Whenever a JSDoc comment is changed - the openAPI doc is not generated from the code itself but from the JSDoc comments

Reply:
I understand, but we're only concerned with the public API, correct? So, we should focus on all */v1/* paths and not the others, or am I missing something?

Reply:
Not sure to understand what you mean - It's basically publishing whatever is present in `public/swagger.json` (the workflow itself doesn't know about our architecture). 

So yes, we only need to add JSDoc in the v1/ files - I'm adding a rule [in this PR](https://github.com/dust-tt/dust/pull/5867) to make sure all /v1 routes are covered

Reply:
Seen IRL, this won't be triggered automatically.